app/templates/application.hbs

```diff
@@ -0,0 +1,4 @@
+<h1>hello</h1>
```

Comment:
I'm assuming you meant to delete this h1 - unless that's an intentional title in which case let's capitalize it.